shardingsphere-mode/shardingsphere-mode-type/shardingsphere-cluster-mode/shardingsphere-cluster-mode-core/src/main/java/org/apache/shardingsphere/mode/manager/cluster/coordinator/registry/cache/subscriber/ScalingRegistrySubscriber.java

```diff
@@ -98,6 +111,10 @@ public void scalingTaskFinished(final ScalingTaskFinishedEvent event) {
             log.info("start to delete cache, ruleCacheId={}", ruleCacheId);
             registryCacheManager.deleteCache(SchemaMetaDataNode.getRulePath(event.getTargetSchemaName()), ruleCacheId);
         }
+        if (schemaNameLockedMap.getOrDefault(decorateLockName(event.getTargetSchemaName()), false)) {
+            log.info("scaling job finished, release schema name lock, event = {}", event);
+            lockRegistryService.releaseLock(event.getTargetSchemaName());
```

Comment:
Seems `schemaNameLockedMap` just has `put` operation, but no `remove` operation, will it cause issue?